tensorflow/core/kernels/quantized_conv_ops.cc

```diff
@@ -492,6 +493,13 @@ class QuantizedConv2DOp : public OpKernel {
     // [ batch, in_rows, in_cols, in_depth ]
     const Tensor& input = context->input(0);
 
+    // validate for non zero dimensions of input.
+    int input_dims = input.shape().dims();
+    for (int i = 0; i < input_dims; ++i) {
+      OP_REQUIRES(context, input.shape().dim_size(i) != 0,
+                  absl::InvalidArgumentError(
+                      "Invalid input: Shapes dimension cannot be 0."));
```

Comment:
```suggestion
    for (int i = 0; i < input_dims; ++i) {
      OP_REQUIRES(context, input.shape().dim_size(i) != 0,
                  absl::InvalidArgumentError(
                      "Invalid input: Shapes dimension cannot be 0."));
    }
```

(missing closing brace)